doc/source/getting-involved.rst

```diff
@@ -45,13 +45,13 @@ Submitting and Merging a Contribution
 
 There are a couple steps to merge a contribution.
 
-1. First rebase your development branch on the most recent version of master.
+1. First merge the most recent version of master into your development branch.
 
    .. code:: bash
 
      git remote add upstream https://github.com/ray-project/ray.git
      git fetch upstream
-     git rebase upstream/master
+     git merge upstream/master
```

Comment:
Is this the same as "git pull . upstream/master"? I always use this instead (haven't even heard of the merge command).

Reply:
ah yeah it is the same but i'l change to yours